.github/workflows/tests.yml

```diff
@@ -71,11 +71,31 @@ jobs:
         env:
           CI: true
 
+  macos:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        node: ['10', '12', '14', '16', '17']
```

Comment:
Are you really sure we need all those versions? Not sure in how far this uses up contingents we get from Github, but also, what bugs do you expect to find here except make CI runs potentially slower? IMO it should be enough to do all relevant versions for the fastest OS (which is Linux on CI) and only do the oldest supported and latest supported for MacOS and Windows.

Reply:
not sure what the GH actions usage limit for OSS is actually. but as far as I know, modules are usually tested against the latest LTS versions on multiple operating systems. but it's all up to the developer to decide.

testing the earliest supported LTS and the latest supported LTS plus latest version is probably fine tho.
since it's free, and just some settings, I'd personally go for all. it might not apply to rollup (or it might), but there are sometimes some small intricacies between node versions.